ketcher-autotests/tests/Structure-Creating-&-Editing/Text-Tool/text-formating.spec.ts

```diff
@@ -0,0 +1,254 @@
+    page,
+  }) => {
+    // Test case: EPMLSOPKET-2260
+    // Checking if possible to put different styles on the created text object
+    await addTextBoxToCanvas(page);
+    await page.getByRole('dialog').getByRole('textbox').fill('TEST123');
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: '13' }).click();
+    await page.getByText('25', { exact: true }).click();
+    await page.getByRole('button', { name: 'bold' }).click();
+    await page.getByRole('button', { name: 'italic' }).click();
+    await page.getByRole('button', { name: 'superscript' }).click();
+    await pressButton(page, 'Apply');
+    await takeEditorScreenshot(page);
+    await page.getByText('TEST123').dblclick();
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: 'subscript' }).click();
+    await pressButton(page, 'Apply');
+  });
+
+  test('Text tool - Save as .ket file', async ({ page }) => {
+    // Test case: EPMLSOPKET-2235
+    // Checking if possible to put different styles on the created text object
+    await addTextBoxToCanvas(page);
+    await page.getByRole('dialog').getByRole('textbox').fill('TEST321');
+    await pressButton(page, 'Apply');
+    await page.getByText('TEST321').dblclick();
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: '13' }).click();
+    await page.getByText('20', { exact: true }).click();
+    await page.getByRole('button', { name: 'bold' }).click();
+    await pressButton(page, 'Apply');
+    await takeEditorScreenshot(page);
+    await saveToFile('ketfile01.ket', 'ketFile');
+    await selectTopPanelButton(TopPanelButton.Clear, page);
+    await selectTopPanelButton(TopPanelButton.Open, page);
+    await openFromFileViaClipboard('tests/test-data/KET/ketfile01.ket', page);
+    await clickInTheMiddleOfTheScreen(page);
+    await takeEditorScreenshot(page);
+    await page.getByText('TEST321').dblclick();
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: 'italic' }).click();
+    await pressButton(page, 'Apply');
+  });
+
+  test('Text tool - Cut/Copy/Paste', async ({ page }) => {
+    // Test case: EPMLSOPKET-2272
+    // Checking if user is able to copy and paste the created text objects
+    await addTextBoxToCanvas(page);
+    await page.getByRole('dialog').getByRole('textbox').fill('TEXT001');
+    await pressButton(page, 'Apply');
+    await page.getByText('TEXT001').click();
+    await selectTopPanelButton(TopPanelButton.Copy, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await page.keyboard.press('Control+v');
+    await page.getByTestId('canvas').click({ position: { x: 250, y: 300 } });
+    await takeEditorScreenshot(page);
+  });
+
+  test('Text tool - Checking if user is able to cut and paste the created text objects', async ({
+    page,
+  }) => {
+    await openFileAndAddToCanvas('KET/text-EPMLSOPKET-2272.ket', page);
```

Comment:
I suggest calling the test files with more understandable names that indicate what kind of structure is being used. So that later, when reading the code, a person can understand what kind of structure to expect on the canvas.